app/ui/browser-modern/model/profile.js

```diff
@@ -12,6 +12,10 @@ specific language governing permissions and limitations under the License.
 
 import Immutable from 'immutable';
 
-export default Immutable.Record({
+import { register } from '../util/record-constructors';
+
+const VERSION = 1;
```

Comment:
Do we have to increment this (and other model's version numbers) everytime any change is made to the state? Should have clear documentation on what the rules are for this so we don't run into a scenario where we're checking for "bad" version numbers of models